pyro/poutine/handlers.py

```diff
@@ -467,26 +467,26 @@ def substitute(  # type: ignore[empty-body]
 
 @overload
 def trace(
-    fn: None = ...,
+    fn: None = None,
     graph_type: Optional[Literal["flat", "dense"]] = None,
     param_only: Optional[bool] = None,
 ) -> TraceMessenger: ...
 
 
 @overload
 def trace(
-    fn: Callable[_P, _T] = ...,
+    fn: Callable[_P, _T],
```

Comment:
I had to remove the default to avoid the mypy error:
```
pyro/poutine/handlers.py:469: error: Overloaded function signatures 1 and 2 overlap with incompatible return types  [overload-overlap]
```

Reply:
Interesting, seems like some weird behavior from mypy